### DISCUSSION TITLE
[Announcement] Isaac Sim 6.0 Early Developer Release for GTC’26

### DISCUSSION THREAD

**Discussion by rthaker01** [isaac-sim/IsaacSim#538] [Announcements]
We’re expanding Isaac Sim 6.0 with updates that enable multiple physics backends, performance improvements, enhanced robot rigging workflows and the ability to operate long-running agents in simulation.

As with other early/experimental functionality, some APIs and schemas may evolve in upcoming updates.

### **Kit 110.0 Upgrade**

Kit 110.0 Upgrade: The latest version of Kit brings support for NVIDIA Omniverse NuRec 3D Gaussian splatting libraries with Fabric Scene Delegate integration, nested rigid body physics with GPU acceleration, and expanded XR support. Full release highlights can be found [here](https://docs.omniverse.nvidia.com/dev-guide/latest/release-notes/110_0_highlights.html#kit-110-0-release-highlights).

### **Physics and multi-backend workflows**

Multiple physics backend support: Isaac Sim now supports multiple physics backends, including both PhysX and Newton. Assets can be authored and simulated in Isaac Sim, and then easily transitioned to Isaac Lab for reinforcement learning without needing to adjust for physics.

Core Experimental API: The previous PyTorch-based Core API is deprecated in favor of a new [Core Experimental API](https://docs.isaacsim.omniverse.nvidia.com/latest/py/docs/overview/experimental.html) built on Warp. This Warp-based implementation is designed as a more future-proof interface for working across multiple physics backends, such as PhysX and Newton.

Updated URDF and MJCF importers: The URDF and MJCF importers have been rebuilt on the USD Exchange SDK. Assets imported through the new importers are automatically structured to be conducive to simulation with multiple physics backends, including PhysX and Newton.

Asset Transformer: Allows users to update legacy robot assets to the new robot asset structure, formatting them in a way that is conducive to simulation with multiple physics backends.

Motion Generation API: New API makes it easier to connect Isaac Sim to external motion planners, with out-of-the-box support for CuMotion.

### **Runtime Architecture Updates**

Asynchronous rendering: Asynchronous rendering support has been added when the simulation is not playing, running simulation and rendering operations on dedicated threads for improved performance and interactivity. Experimental support for asynchronous rendering when the simulation is playing has also been added, which could potentially provide improved performance, interactivity, and decoupling of platform components such as physics simulation and rendering.

Fabric support: Isaac Sim now has full support for Fabric for transformations, improving simulation performance by using a vectorized representation of USD data for runtime operations.

### **Robot authoring and debugging tools**

Robot rigging tools aimed at faster iteration and easier debugging:

Robot Inspector: Visualize the robot based on its hierarchy or as a flat list of links, isolated from the remaining clutter of the asset, And display the kinematic tree on the highlighted robot.

Debugging Tools: In the robot Inspector, debugging tools to temporarily anchor the robot in place, and disable or bypass subsections of the robot to help isolate where authoring issues may be coming from

Self-Collision Detection: Tool to search and highlight components of the robot that are in collision with itself.

Robot Poser: A jogging tool that lets you move robots independently of the simulation and save robot poses.

### **ROS2**

ROS2 Jazzy native sourcing: ROS2 Jazzy system-level installations can now be sourced directly with Isaac Sim, enabled by full Python 3.12 support. Previously, users were limited to internal libraries due to Python 3.11 constraints.

ROS2 compressed image publishing: Isaac Sim now supports ROS2 H.264 compressed RGB publishing with hardware-accelerated encoding, improving performance for software-in-the-loop testing workflows.

Point cloud metadata: RTX LiDAR metadata, such as intensity, can now be included in ROS2 PointCloud2 messages, allowing users to select which metadata fields they would like to publish

Experimental ROS2 distribution support: Any ROS2 distribution built with Python 3.12 can now be sourced and used with Isaac Sim, extending support beyond Jazzy to additional ROS2 distros.

### **Asset ecosystem and distribution**

SimReady asset catalog: A new curated collection of SimReady assets is available in the Content Browser, including robot models from FANUC and Comau, as well as a variety of props. These assets have passed both USD validation and runtime tests for improved out-of-the-box reliability, and the catalog will expand significantly in future releases.

### **Action and Event Data Generation**

Action and event data generation: Adds behavior planning, expanded incident simulation, and broader platform flexibility through improved tooling.

Agent simulation and unified pipeline: Isaac Sim Replicator Agent now supports a character USD schema for cross-scene portability. A new UI makes it easier to configure agent simulations, allows users to spawn characters, issue behaviors, and produce both annotated sensor data and VLM captions for the  simulation run.

Object simulation and physics: Isaac Sim Replicator Object now supports randomization of physics attributes (mass, friction, restitution) across scene objects and allows users to apply new placement strategies (pyramid stacking, world/local-space force application) for synthetic data generation.

Isaac Sim Replicator Object: Chat IRO now supports multiple generative AI models (e.g. Llama-4, ChatGPT-OSS, Qwen3) and direct rendering of 3D scenes. Users can now iteratively generate complex 3D scene randomizations using natural language prompts.

VLM scene captioning: Isaac Sim Replicator Captions allows for region-of-interest based caption control and configurable LLM prompts. Users can now produce more accurate training data for Vision-Language Models using simulation-label-driven captions.

FlowUSD fire and smoke writer: A new writer allows users to capture and generate volumetric fire and smoke data in addition to standard RGB, depth, semantic segmentation outputs.

RTX Sensor Placement & Calibration: Users can now randomize camera placement to maximize coverage for a defined number of cameras. A new API makes it easier to calibrate sensors by providing direct access to camera intrinsics, extrinsics, and field-of-view coverage visualization.

### **Get started**

Download the source code for the early developer release of [Isaac Sim 6.0](https://github.com/isaac-sim/IsaacSim/tree/develop) from [GitHub](https://github.com/isaac-sim/IsaacSim).

Connect with the community and submit feedback on Isaac Sim 6.0 on [Discord](https://discord.com/channels/827959428476174346/1383208629532098610) or [GitHub Issues](https://github.com/isaac-sim/IsaacSim/issues).
